mod/consensus/pkg/cometbft/service/abci.go

```diff
@@ -57,7 +59,6 @@ func (s *Service[LoggerT]) InitChain(
 
 	// On a new chain, we consider the init chain block height as 0, even though
 	// req.InitialHeight is 1 by default.
-	initHeader := cmtproto.Header{ChainID: req.ChainId, Time: req.Time}
```

Comment:
`initHeader` is really used only in `defer` call, to prepare context for next block. Moved it to defer to simplify logic and reduce its scope